.github/workflows/build.yml

```diff
@@ -19,15 +19,24 @@ jobs:
         machine:
           - qcs6490-rb3gen2-core-kit
           - qcs6490-rb3gen2-vision-kit
+          - qcs8275-iq-8275-evk
           - qcs8300-ride-sx
-          - qcs9075-rb8-core-kit
+          - qcs9075-iq-9075-evk
           - qcs9075-ride-sx
           - qcs9100-ride-sx
         bsp:
           - custom
         include:
+          - machine: qcs615-adp-air
+            bsp: base
+          - machine: qcs8275-iq-8275-evk
+            bsp: base
           - machine: qcs8300-ride-sx
             bsp: base
+          - machine: qcs9075-iq-9075-evk
+            bsp: base
+          - machine: qcs9075-ride-sx
+            bsp: base
```

Comment:
Do we need base for all the combinations here? I'm not sure it was tested for all targets.

Reply:
In theory all supported.

Reply:
well, the release notes might not be so correct. build failed for qcs8275-iq-8275-evk.conf base config, missing DTB..